linetools/isgm/abscomponent.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, radec, Zion, z, vlim, Ej=0./u.cm, A=None, Ntup=None, comment=
         self.name = ions.ion_name(self.Zion, nspace=1)
         self._abslines = []
 
-    def add_absline(self, absline, tol=0.1*u.arcsec):
+    def add_absline(self, absline, tol=0.1*u.arcsec, skip_vel=False):
```

Comment:
This seems very risky to me; please add a warning message. Wouldn't it be better to have a velocity tolerance which by defaults is that of the Component itself, but that can also be set to arbitrary large values or None (which is effectively a skip_vel = True).